manageiq-operator/pkg/helpers/miq-components/postgresql.go

```diff
@@ -40,81 +44,86 @@ func postgresqlSecretName(cr *miqv1alpha1.ManageIQ) string {
 	return secretName
 }
 
-func NewPostgresqlConfigsConfigMap(cr *miqv1alpha1.ManageIQ) *corev1.ConfigMap {
+func NewPostgresqlConfigsConfigMap(cr *miqv1alpha1.ManageIQ) (*corev1.ConfigMap, controllerutil.MutateFn) {
 	labels := map[string]string{
 		"app": cr.Spec.AppName,
 	}
-	return &corev1.ConfigMap{
+
+	data := map[string]string{
+		"01_miq_overrides.conf": postgresqlOverrideConf(),
+	}
+
+	configMap := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "postgresql-configs",
 			Namespace: cr.ObjectMeta.Namespace,
-			Labels:    labels,
-		},
-		Data: map[string]string{
-			"01_miq_overrides.conf": postgresqlOverrideConf(),
 		},
 	}
+
+	mutateFn := configMapMutateFn(configMap, labels, data)
```

Comment:
Hm, I'm wondering if we don't actually want to enforce the contents here to allow for user modification... Probably okay for now, but worth considering.